build/template_kql/common/calculate_score.kql

```diff
@@ -19,10 +19,12 @@ extend ReliabilityScore = case(
     Type == 'microsoft.apimanagement/service', (OtherSku + Gt1CapacityCount) * 100 / (2 * ResourceTotal),
     // Storage - 2 checks // score = ( "kind" + "replication type") * 100 / (2 * ResourceTotal)    //********* Future - "access tier hot"
     Type == 'microsoft.storage/storageaccounts' or Type == 'microsoft.classicstorage/storageaccounts', (V2StorageEnabled + RAStorageEnabled) * 100 / (2 * ResourceTotal),
-    // AzureFirewall - 1 check // score = (AZ) * 100 / (1 * ResourceTotal)
-    Type == 'microsoft.network/azurefirewalls', (AvZoneCount + NAAvZoneCount)* 100 / ResourceTotal,
+    // AzureFirewall - 1 check // score = (State + AZ) * 100 / (2 * ResourceTotal)
+    Type == 'microsoft.network/azurefirewalls', (AzFWSucceededStateCount + AvZoneCount + NAAvZoneCount)* 100 / (2 * ResourceTotal),
     // AFD - 1 check // score = state  * 100 / (1 * ResourceTotal)
     Type == 'microsoft.network/frontdoors', AFDEnabledStateCount * 100 / ResourceTotal,
+    // CDN - 1 check // score = state  * 100 / (1 * ResourceTotal)
+    Type == 'microsoft.cdn/profiles', CDNEnabledStateCount * 100 / ResourceTotal,
```

Comment:
(optional) Can be included in AFD itself since now it is one item and old AFD is classic SKU

Eg:  Type == 'microsoft.network/frontdoors' or Type == 'microsoft.cdn/profiles', (AFDEnabledStateCount + CDNEnabledStateCount ) * 100 / ResourceTotal,

Reply:
It's more optimized than the current one. For further detail, please see following new commit.
https://github.com/tsubasaxZZZ/reliability-workbook/commit/5ff71d2bd19233d367871dc4ad80a2607952cf63